packages/jest-jasmine2/src/PCancelable.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/* eslint-disable no-this-before-super, constructor-super */
+
+class CancelError extends Error {
```

Comment:
you can look at https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/p-cancelable/index.d.ts for inspiration.

That said, this is essentially a vendored file (we need to control the `Promise` global, which is why it's in our source code), so just keeping it as `js` and ignoring the import should be fine